README.md

```diff
@@ -1,229 +1,144 @@
-(iii) Updates secret-conf.properties file with default keystore and callback class. 
+Reporting Product Issues
+========================
+All known issues of WSO2 IS are filed at: https://github.com/wso2/product-is/issues. Please check this list before opening a new issue.
 
-cipher-tool.properties, cipher-text.properties and secret-conf.properties files 
-			can be found at repository/conf/security directory. 
+#### Opening an issue
+Help us make our software better! Submit any bug reports or feature requests through GitHub: https://github.com/wso2/product-is/issues.
 
-2. Start server by running wso2server script from bin directory
+#### Reporting Security Issues
+Please do not report security issues via GitHub issues. Instead, follow the WSO2 Security Vulnerability Reporting Guidelines.
 
-> wso2server.sh   (in UNIX)
 
-By default mode, it would ask you to enter the master password 
-(By default, master password is the password of carbon keystore and private key) 
+Join our community!
+========================
+- Read our documentation at [https://is.docs.wso2.com/en/latest/](https://is.docs.wso2.com/en/latest/).
+- Get help on [Stack Overflow](https://stackoverflow.com/questions/tagged/wso2-is).
+- Join the conversation on [Discord](https://discord.gg/wso2).
+- Learn more by reading articles from our [library](https://wso2.com/library/).
 
-3. Change any password by running ciphertool script from bin directory.  
 
-> ciphertool -Dchange  (in UNIX)
 
-For more details see
-https://is.docs.wso2.com/en/7.1.0/deploy/security/encrypt-passwords-with-cipher-tool/
 
-Support
-=======
-We are committed to ensuring that your enterprise middleware deployment is completely supported from
-evaluation to production. Our unique approach ensures that all support leverages our open development
-methodology and is provided by the very same engineers who build the technology.
 
-For more details and to take advantage of this unique opportunity, visit http://wso2.com/support/.
+Commercial Support
+==================
+You can take advantage of a WSO2 on-prem product subscription for the full range of software product benefits like expert support, continuous product updates, vulnerability monitoring, and access to the licensed distribution for commercial use.
 
+To learn more, check WSO2 Subscription.
 
-For more information on WSO2 Carbon, visit the WSO2 Oxygen Tank (http://wso2.org)
 
-For more information about WSO2 Identity Server please see http://wso2.org/projects/identity or visit the
-WSO2 Oxygen Tank developer portal for addition resources.
 
-For further details, see the WSO2 Carbon documentation at
-https://docs.wso2.com/display/Carbon4411/WSO2+Carbon+Documentation
+Contributing
+===============
+If you are planning on contributing to the development efforts of WSO2 <Product Name>, you can do that by checking out the latest development version. The master branch holds the latest unreleased source code.
```

Comment:
```suggestion
If you are planning on contributing to the development efforts of WSO2 Identity Server, you can do that by checking out the latest development version. The master branch holds the latest unreleased source code.
```

Reply:
@pavinduLakshan thanks for catching this.